ceph.spec.in

```diff
@@ -2132,6 +2132,7 @@ fi
 %{_bindir}/rgw-gap-list-comparator
 %{_bindir}/rgw-orphan-list
 %{_bindir}/rgw-policy-check
+%{_bindir}/rgw-restore-bucket-index
```

Comment:
QE is expecting these scripts to be installed along with `radosgw-admin`, which is packaged in the `%files common` section. the `radosgw` package is generally only installed on the radosgw server nodes

Reply:
Thanks, @cbodley. As you know (but documenting for the PR here), there's an add-on PR to address this: https://github.com/ceph/ceph/pull/50545.